openedx/core/lib/api/permissions.py

```diff
@@ -153,7 +153,9 @@ def has_object_permission(self, request, view, obj):
 
     def has_permission(self, request, view):
         user = request.user
+        # TODO Update tests
         return user.is_staff \
             or (user.username == request.GET.get('username')) \
             or (user.username == getattr(request, 'data', {}).get('username')) \
-            or (user.username == getattr(view, 'kwargs', {}).get('username'))
+            or (user.username == getattr(view, 'kwargs', {}).get('username')) \
+            or (user.id == getattr(request, 'data', {}).get('user'))
```

Comment:
I'm confused why it's in a core permissions app? Isn't that particular data format specific to the experiments app?